manifests/config.pp

```diff
@@ -0,0 +1,169 @@
+# == Class nzbget::config
+#
+# This class is called from nzbget for service config.
+#
+
+class nzbget::config {
+  if ($::nzbget::manage_service_dirs[0]) {
```

Comment:
What the fuck is this line trying to say (and the others like it below. I assume something like:

```suggestion
  if ($::nzbget::manage_service_dirs.length > 0) {
```